### PR TITLE
feat: added action to useKeyboard

### PR DESCRIPTION
**Pull request by giuliano176**

### CODE REVIEW

File: src/components/display/List.tsx [hunk 1]
```diff
@@ -66,18 +66,26 @@ const StyledDiv = styled.div<{
 		)};
 `;
 
+type ItemProps = {
+	id: string;
+	onClick: (evt: unknown) => void;
```
Comment (beawar) [inline src/components/display/List.tsx]: Why unknown and not a generic Event?

Comment (giuliano176) [reply src/components/display/List.tsx]: I agree, Event is a better fit

Comment (gnekoz) [reply src/components/display/List.tsx]: Also, could the onClick prop be optional?

**Comment by beawar** [comment]
I removed the "onClick" and "disabled" props spreading since they are easily accessible from the item itself directly in the ItemComponent build from outside. See List.test.tsx for an example

**Review by CataldoMazzilli — APPROVED** [review]

**Pull request merged**